bigquery_etl/cli/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""bigquery-etl CLI."""
+
+import click
+
+from bigquery_etl.cli.query import query_cli
+from bigquery_etl.version import __version__
+
+
+def cli():
+    """Create the bigquery-etl CLI."""
+    commands = {
+        "query": query_cli,
```

Comment:
This is where more commands that are defined in separate files can be added over time.